Formatting updates (#971)
spatialos/UnrealGDK: SpatialGDK/Documentation/content/tutorials/deployment-manager/tutorial-deploymentmgr-play.md

# Multiple deployments for session-based games
## Step 6: Play your game

> 1.  select **session_0** from the deployment list. This opens the deployment overview screen.

Reviewer: ```suggestion
1.  Select **session_0** from the deployment list. This opens the deployment overview screen.
```

> 2. select **LAUNCH** on the left of the page.<br/></br>

Reviewer: ```suggestion
2. Select **LAUNCH** on the left of the page.<br/></br>
```

> ![img]({{assetRoot}}assets/deployment-manager/deploymentmgr-consoleoverview.png)

Author: ```suggestion
![img]({{assetRoot}}assets/deployment-manager/deploymentmgr-consoleoverview.png)</br>
```

_Image: The SpatialOS Console with the game client **LAUNCH** button highlighted._<br/></br>

> 3. select the **Launch** button that appears in the center of the page to open the [SpatialOS Launcher](https://docs.improbable.io/reference/13.7/shared/operate/launcher). The Launcher automatically downloads a game client for this deployment and runs it on your local machine. </br></br>

Reviewer: ```suggestion
3. Select the **Launch** button that appears in the center of the page to open the [SpatialOS Launcher](https://docs.improbable.io/reference/latest/shared/operate/launcher). The Launcher automatically downloads a game client for this deployment and runs it on your local machine. </br></br>
```

![img]({{assetRoot}}assets/deployment-manager/deploymentmgr-launch.png)
_Image: The SpatialOS Console **Launch a client** pop-up window._<br/></br>

> 4. Once the game client has launched, you see the Example Project start screen.</br>

Reviewer: ```suggestion
1. Once the game client has launched, you see the Example Project start screen.</br>
```
